src/test/java/AddNewContactTest.java

```diff
@@ -0,0 +1,44 @@
+import org.testng.annotations.Test;
+
+import static java.lang.Thread.sleep;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class AddNewContactTest extends BaseTest {
+
+
+    @Test
+    public void successAddNewContact() throws InterruptedException {
+        LoginHelper loginHelper = new LoginHelper(app.driver);
+        loginHelper.loginTestHelper();
+        ContactsPage contactsPage = new ContactsPage(app.driver);
+        sleep(2000);
+        contactsPage.clickButtonAddNewContact();
+        assertTrue(contactsPage.modalWindowIsDisplayed());
+        AddContactDialog addContactDialog = new AddContactDialog(app.driver);
+        addContactDialog.enterFirstName("Maria");
```

Comment:
Here you can use variables for first name, last name and description, but not necessarily.

Reply:
I prefer this variation

Reply:
I think this may be required when scaling the autotest.